views/welcome_wizard.html

```diff
@@ -3,8 +3,11 @@
 <% block content %>
 <script>
   var mailpoet_logo_url = '<%= cdn_url('welcome-wizard/mailpoet-logo.20190109-1400.png') %>';
-  var email_course_illustration = '<%= cdn_url('welcome-wizard/wizard-email-course.20190109-1400.png') %>';
-  var woocommerce_screenshot_url = '<%= cdn_url('welcome-wizard/woocommerce-auto-emails-tilted.20181121-1440.png') %>';
+  var step_1_illustration_url = '<%= cdn_url('welcome-wizard/step1.20190409.png') %>';
```

Comment:
Could you please use some names related to step (maybe names similar to step names) instead of numbers? We change wizard quite often and when we change order of steps a semantic name will be handy.